external/ag-website-shared/src/components/footer/Footer.tsx

```diff
@@ -19,7 +19,11 @@ const MenuColumns = ({ footerItems }: { footerItems: FooterItem[] }) =>
             <ul className="list-style-none">
                 {links.map(({ name, url, newTab, iconName }: any) => (
                     <li key={`${title}_${name}`}>
-                        <a href={urlWithBaseUrl(url)} {...(newTab ? { target: '_blank', rel: 'noreferrer' } : {})}>
+                        <a
+                            id={`${name.toLowerCase().replace(' ', '-')}-nav`}
```

Comment:
you can also use `github-slugger` here eg,https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-website/src/components/gallery/components/GallerySeriesLink.jsx#L17